__init__.py

```diff
@@ -81,6 +86,12 @@ def execute(self, context):
             # Copy properties to settings
             settings = self.as_keywords(ignore=("filter_glob",))
 
+            # Calculate a global matrix to apply to each mesh.
+            settings['global_matrix'] = axis_conversion(
```

Comment:
This makes for an interesting setting. I think it may be more usable if we had the forward and up vectors as settings, and then used orientation_helper_factor in blendergltf.py to create the matrix. That would at least work better for the most common use cases, but it could be a bit limiting. I'll need to think about this some more.